frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -181,13 +184,54 @@ export async function importTracingFiles(files: Array<File>, createGroupForEachF
       }
     };
 
+    const tryParsingFileAsZip = async file => {
+      try {
+        const zipFile = await JSZip().loadAsync(readFileAsArrayBuffer(file));
+        const nmlFileName = Object.keys(zipFile.files).find(
+          key => _.last(key.split(".")) === "nml",
+        );
+        const nmlFile = await zipFile.file(nmlFileName).async("blob");
+        const nmlImportActions = await tryParsingFileAsNml(nmlFile);
+
+        const dataFileName = Object.keys(zipFile.files).find(
+          key => _.last(key.split(".")) === "zip",
```

Comment:
same here. maybe extract this into a helper function à la `findFileNameWithExtension`.